docs/vocascan-server/traefik/traefik.md

```diff
@@ -0,0 +1,149 @@
+# Installation with Traefik
+
+This guide will get you through the installation of your own vocascan-server paired with Traefik as a reverse proxy and
+Let's Encrypt.
+
+1. First you have to install Docker and Docker-Compose. To do this have a look at point 1 and 2 our
+   [guide](vocascan-server/docker/docker) in the general vocascan-server installation section.
+
+2. After that create a secure password hash for the Traefik monitoring website with
+   `htpasswd`
+
+   ```
+   docker run --entrypoint htpasswd httpd:2 -Bbn admin your_password
+   ```
+
+   The output will look something like this. Store it for the later configuration 
+
+   ```
+   admin:$apr1$ppHrFJc4$3/iv8jBNSDXlxiAuP1GDH1
+   ```
+
+3. Download the vocascan-server repository
+
+   ```bash
+   git clone https://github.com/vocascan/vocascan-server.git
+   ```
+
+4. Copy Traefik and Docker configs from cloned repo
+
+   copy Traefik folder in your current directory
+
+   ```bash
+   cp -R vocascan-server/docker/traefik traefik
+   ```
+
+   Change directory
+
+   ```bash
+   cd traefik
+   ```
+
+5. Create `acme.json` to store certificate information
+
+   ```bash
+   touch acme.json
+   ```
+
+   - make file only accessable by root
+
+   ```bash
+   chmod 600 acme.json
+   ```
+
+6. Open `vocascan.config.js` to configure your server
+
+   ```bash
+   nano vocascan.config.js
+   ```
+
+   For more details about the configuration see [configuration](vocascan-server/configuration) page.
+
+7. Open `traefik.toml`
+
+   ```bash
+   nano traefik.toml
+   ```
+
+   and update the email prop to your email
+
+   ```
+                        ...
+   14    [certificatesResolvers.lets-encrypt.acme]
+   15       email = "your@mail.com"
+   16       storage = "acme.json"
+                        ...
+   ```
+
+8. Open `traefik_dynamic.toml`
+
+   ```bash
+   nano traefik_dynamic.toml
+   ```
+
+   - Add the secret key (line 3), you created in the beginning (step 2)
+
+   ```
+   1    [http.middlewares.simpleAuth.basicAuth]
+   2       users = [
+   3          "admin:secret_key"
+   4    ]
+                        ...
+   ```
+
+   - Update your domain (line 7)
+
+   ```
+                        ...
+   6    [http.routers.api]
+   7       rule = "Host(`monitor.your_domain.com`)"
+                        ...
+   ```
+
+9. Open `docker-compose.yml`
+
+   ```bash
+   nano docker-compose.yml
+   ```
+
+   update your docker variables and replace your domain
```

Comment:
sorry to be precise here, but these are docker labels. If you try to search for `docker variables` on the internet, you wont find the right thing.

Reply:
Resolved, as this sentence is not existing anymore after some changes